cmd/gentx.go

```diff
@@ -19,8 +19,9 @@ var gentxCmd = &cobra.Command{
 		senderCount, _ := cmd.Flags().GetInt("sender-count")
 		txCount, _ := cmd.Flags().GetInt("tx-count")
 		txStoreDir, _ := cmd.Flags().GetString("tx-store-dir")
+		mempool, _ := cmd.Flags().GetInt("mempool")
```

Comment:
maybe rename this to `mempool-size` or something that's more descriptive?

Reply:
I agree. 

This command-line option (mempool) was already used in the previous commit. We can update all occurrences of this option name in a separate commit.